src/content/changelog/4-78-3-2020-10-20-2020.md

```diff
@@ -0,0 +1,12 @@
+---
+title: '4.78.3: 2020-10-20'
+date: 2020-10-20T04:00:00.000Z
+version: 4.78.3
+changelog:
+  - API
+---
+
+### Changed
+- The [API v4 spec](http://www.linode.com/docs/api) has been moved to reside in the new [Linode Docs site](http://www.linode.com/docs). The links within the spec file have been updated to reflect this new location.
```

Comment:
One suggestion:
* Linode's [API v4 documentation ](https://www.linode.com/docs/api) has been moved to reside in the new [Linode Docs site](http://www.linode.com/docs). 
  * Our OpenAPI specification can now be downloaded at [https://www.linode.com/docs/api/openapi.yaml](https://www.linode.com/docs/api/openapi.yaml)
* Any links within the spec file have been updated to reflect this new location.

Reply:
updated - thank you :)